action/messageHubFeed.js

```diff
@@ -14,7 +14,7 @@ function main(params) {
     var namespace = encodeURIComponent(triggerComponents[1]);
     var trigger = encodeURIComponent(triggerComponents[2]);
 
-    var feedServiceURL = 'http://169.46.145.194:5000/triggers/' + namespace + '/' + trigger;
+    var feedServiceURL = 'http://' + params.package_endpoint + '/triggers/' + namespace + '/' + trigger;
```

Comment:
It is expected that this parameter will be bound when the action is installed. See #34.